jira-feature-collector/src/main/java/com/capitalone/dashboard/util/ClientUtil.java

```diff
@@ -241,6 +255,79 @@ public JSONObject toCanonicalSprintJSON(String nativeRs) {
 
 		return canonicalSprint;
 	}
+	
+	public List<Sprint> parseSprints(Object data) throws ParseException {
+		List<Sprint> sprints = new ArrayList<>();
+		
+		if (data instanceof JSONArray) {
+			for (Object obj : (JSONArray)data) {
+				String rawToString = obj != null? obj.toString() : null;
+				
+				Sprint sprint = parseSprint(rawToString);
+				
+				sprints.add(sprint);
+			}
+		} else if (data instanceof org.codehaus.jettison.json.JSONArray) {
+			org.codehaus.jettison.json.JSONArray jsonA = (org.codehaus.jettison.json.JSONArray)data;
+			for (int i = 0; i < jsonA.length(); ++i) {
+				Object obj;
+				try {
+					obj = jsonA.get(i);
+				} catch (JSONException e) {
+					throw new RuntimeException("", e);
+				}
+				
+				String rawToString = obj != null? obj.toString() : null;
+				
+				Sprint sprint = parseSprint(rawToString);
+				
+				sprints.add(sprint);
```

Comment:
Looks like it. I would rather add an empty sprint then get a NPE or ruin the collector